### ISSUE TITLE
Add lead in time when testing a beatmap in the editor

### ISSUE THREAD

**Issue by peppy** [ppy/osu#24607]
This is done in stable, so we should match that behaviour as a starting point.

### Discussed in https://github.com/ppy/osu/discussions/24437

<div type='discussions-op-text'>

<sup>Originally posted by **Craanky** August  1, 2023</sup>
It instantly starts in Osu! which makes you miss the first few notes, so it would be nice if it starts like Normal beatmaps do, Quaver does that as an example. It also lags quite a lot for me.</div>

**Comment by ZenoG2** [comment]
Should probably be converted to a discussion as there is now the 3 2 1 countdown.

**Comment by peppy** [comment]
Is there? I just checked and there's no forced countdown when entering test mode yet.

**Comment by ZenoG2** [comment]
No, meant to say that when the discussion was opened the game didn't have a countdown, but now it does, so should be converted to discussion as it's not gonna be ported like it was on Stable (add a delay).
